packages/core/src/index.ts

```diff
@@ -63,7 +63,10 @@ export { hasSpansEnabled } from './utils/hasSpansEnabled';
 export { isSentryRequestUrl } from './utils/isSentryRequestUrl';
 export { handleCallbackErrors } from './utils/handleCallbackErrors';
 export { parameterize, fmt } from './utils/parameterize';
-export { addAutoIpAddressToSession, addAutoIpAddressToUser } from './utils/ipAddress';
+
+export { addAutoIpAddressToSession } from './utils/ipAddress';
```

Comment:
do we leave this on purpose iun-deprecated - is this still needed?

Reply:
Yes, for now, let's leave it undeprecated. Checked with @cleptric and @Dav1dde and there are no PII-related concerns. The reason being that there's no special-casing Relay-side for JS that in the absence of `'{{auto}}'` IP addresses are still inferred.